### PR TITLE
Start converting to log module instead of Println

### PR DESCRIPTION
**Pull request by ysangkok**
In this PR are only logging categories for CSN and Pollard, but more can be added if you think the approach is OK.

### CODE REVIEW

**Comment by kcalvinalvin** [comment]
Getting a build error with this log on my local machine and github actions confirms this.

```
../utreexo/pollardutil.go:8:2: cannot find package "github.com/mit-dci/utreexo/log" in any of:
	/usr/local/Cellar/go/1.13.6/libexec/src/github.com/mit-dci/utreexo/log (from $GOROOT)
	/Users/timapple/go/src/github.com/mit-dci/utreexo/log (from $GOPATH)
```

**Comment by ysangkok** [comment]
@kcalvinalvin  Yeah, I dunno if I am doing something wrong, but the log module needs to be available to utreexo in the future too, so that's why it is on the level with cmd and utreexo. You can build by running `go build` in side the `cmd` (as your working directory)

**Comment by kcalvinalvin** [comment]
> You can build by running `go build` in side the `cmd` (as your working directory)

I should note that I did this when testing and go gave the build error.

File: utreexo/pollardutil.go [hunk 1]
```diff
@@ -5,6 +5,7 @@ import (
 	"os"
 
 	"github.com/mit-dci/utreexo/cmd/util"
+	"github.com/mit-dci/utreexo/log"
```
Comment (kcalvinalvin) [inline utreexo/pollardutil.go:8]: Did you add a package to the repo? In the commit, there is no log/ directory

Comment (ysangkok) [reply utreexo/pollardutil.go:8]: Oh, right, that explains the failure too. Sorry, dunno how I could have missed this.

**Pull request closed without merging**